README.md

```diff
@@ -76,7 +76,7 @@ Or on CodeSandBox at the following url: <a href="https://codesandbox.io/s/04wxlo
 | autoplayReverse            | `React.PropTypes.bool`                                                                                                                                                                                                                                                 | Only meaningful when `autoplay` is already true. When `autoplayReverse` is also true, autorotation cycles through slides indexes from high to low.                                                                                                                                          | `false`                                                                                                            |
 | beforeSlide                | `React.PropTypes.func`                                                                                                                                                                                                                                                 | Hook to be called before a slide is changed                                                                                                                                                                                                                                                 |                                                                                                                    |
-| cellAlign                  | `React.PropTypes.oneOf(['left', 'center', 'right'])`                                                                                                                                                                                                                   | When displaying more than one slide, sets which position to anchor the current slide to. When `cellAlign='center'` or `cellAlign='right'`, keep `slidesToScroll` prop set to default.**Is overridden to `left` when `transitionMode="fade"`**                                               |                                                                                                                    |
+| cellAlign                  | `React.PropTypes.oneOf(['left', 'center', 'right'])`                                                                                                                                                                                                                   | When displaying more than one slide, sets which position to anchor the current slide to. **Is overridden to `left` when `transitionMode="fade"`**                                               |                                                                                                                    |
```

Comment:
We no longer need to warn users about setting `slidesToScroll` to the default value, should work with whatever they type in :D

Reply:
Awesome!!